src/GitHub/Data/PullRequests.hs

```diff
@@ -160,6 +159,9 @@ data PullRequestEventType
     | PullRequestUnassigned
     | PullRequestLabeled
     | PullRequestUnlabeled
+    | PullRequestReviewRequested
```

Comment:
added these because the API docs list them.

Not sure if it makes sense to make a catch-all `OtherPullRequestEventType Text` which might make the library a little more future-proof.